[docs] Warning about Istio and admission-policy-engine restrictions (#7398)
deckhouse/deckhouse: modules/015-admission-policy-engine/docs/README.md

{% alert level="info" %}
The module does not apply policies to system namespaces.

> In deckhouse version >= 1.55. Istio sidecar-injector container starts only in `Privileged` mode

Reviewer: ```suggestion
When using Deckhouse version >= 1.55, the Istio `sidecar-injector` container runs only in `Privileged` mode.
```

{% alert level="info" %}
The module does not apply policies to system namespaces.

> When using Deckhouse version >= 1.55, the Istio `sidecar-injector` container runs only in `Privileged` mode.

Reviewer 2: ```suggestion
When using Deckhouse version >= 1.55, the module works in `Baseline` mode by default. In this regard, Istio's init-containers can't configure rules for application's traffic interception, so the Pod's can't start. The solution is either configure the module to work in `Restricted` mode or add a label `security.deckhouse.io/pod-policy=privileged` to application's Namespace.
```
